server/src/main/java/fr/uga/l3miage/tp4/models/TestCenterEntity.java

```diff
@@ -0,0 +1,25 @@
+package fr.uga.l3miage.tp4.models;
+
+import fr.uga.l3miage.tp4.enums.TestCenterCode;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class TestCenterEntity {
+    @Id
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    private TestCenterCode code;
+    private String university;
+    private String city;
+
+    @OneToMany(mappedBy = "test_Center")
+    private Set<CandidateEntity> candidates;
+
```

Comment:
Non dans la classe candidat tu à déclaré un `TestCenterEntity testCenterEntity ` et non un `TestCenterEntity test_Center `, et d'ailleurs on n'écrit pas de `snake_case` en java

```suggestion
    @OneToMany(mappedBy = "testCenterEntity")
    private Set<CandidateEntity> candidates;
```